tests/conftest.py

```diff
@@ -73,19 +80,31 @@ def migrated_database(self):
                 engine.dispose()
 
 
-@pytest.fixture(scope='session')
+@pytest.yield_fixture(scope='session')
 def _redshift_database_tool():
     from rs_sqla_test_utils import models
-    if 'PGPASSWORD' in os.environ:
+    if 'PGPASSWORD' not in os.environ:
+        pytest.skip('This test will only work on Travis.')
+
+    session = requests.Session()
+    resp = session.post('https://bigcrunch.herokuapp.com/sessions/')
```

Comment:
I know, I didn't request the index and traverse my REST API. (but I could have, and that's the point)